ext/gtest/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if (WIN32)
 		BINARY_DIR "${PROJECT_BINARY_DIR}/gtest"
 
 		INSTALL_DIR "${PROJECT_BINARY_DIR}/lib"
-		INSTALL_COMMAND "${CMAKE_COMMAND}" -E copy ${PROJECT_BINARY_DIR}/gtest/libgtest.a ${PROJECT_BINARY_DIR}/gtest/libgtest_main.a ${PROJECT_BINARY_DIR}/lib
```

Comment:
1. Setting BINARY_DIR to "${PROJECT_BINARY_DIR}/lib" is enough. No need to use another install command.
2. If cmake is installed in a directory with blanks(e.g. on Windows), the INSTALL_COMMAND cannot work without ".